common/src/com/elikill58/negativity/common/protocols/GroundSpoof.java

```diff
@@ -0,0 +1,114 @@
+package com.elikill58.negativity.common.protocols;
+
+import com.elikill58.negativity.api.block.Block;
+import com.elikill58.negativity.api.block.BlockFace;
+import com.elikill58.negativity.api.entity.Entity;
+import com.elikill58.negativity.api.entity.Player;
+import com.elikill58.negativity.api.events.EventListener;
+import com.elikill58.negativity.api.events.Listeners;
+import com.elikill58.negativity.api.events.player.PlayerMoveEvent;
+import com.elikill58.negativity.api.item.Materials;
+import com.elikill58.negativity.api.location.Location;
+import com.elikill58.negativity.universal.Cheat;
+import com.elikill58.negativity.universal.Negativity;
+import com.elikill58.negativity.universal.report.ReportType;
+
+import java.util.EnumSet;
+
+public class GroundSpoof extends Cheat implements Listeners {
+    private static final EnumSet<BlockFace> totalFaces = EnumSet.of(BlockFace.WEST, BlockFace.EAST, BlockFace.SOUTH,
+            BlockFace.NORTH, BlockFace.NORTH_WEST, BlockFace.SOUTH_WEST, BlockFace.NORTH_EAST, BlockFace.SOUTH_EAST);
+
+    public GroundSpoof() {
+        super("GROUND_SPOOF", CheatCategory.MOVEMENT, Materials.DIRT, false, false, "groundspoof");
+    }
+
+    @EventListener
+    public void onGroundSpoof(PlayerMoveEvent e) {
+        if (!((Entity) e.getPlayer()).isOnGround()) {
```

Comment:
You could move this condition inside the `isOnGround` static method you use in the next if statement.

Reply:
what do you mean?

Reply:
Move it into "isOnGround" static method (declared line 53)